core/assets/bundles/bundle_fr.properties

```
@@ -1125,12 +1125,12 @@ setting.coreitems.name = Afficher les objets du Noyau
 setting.position.name = Afficher la position du joueur
 setting.mouseposition.name = Afficher la Position de la Souris
 setting.musicvol.name = Volume de la Musique
-setting.atmosphere.name = Montrer l'Atmosphère de la planète
+setting.atmosphere.name = Montrer l'Atmosphère des planètes
 setting.drawlight.name = Dessiner les Ombres/Lumières
 setting.ambientvol.name = Volume Ambiant
 setting.mutemusic.name = Couper la Musique
-setting.sfxvol.name = Volume des Sons et Effets
-setting.mutesound.name = Couper les Sons et Effets
+setting.sfxvol.name = Volume des Sons et des Effets
+setting.mutesound.name = Couper les Sons et les Effets
```

```suggestion
setting.sfxvol.name = Volume des Sons et Effets
setting.mutesound.name = Couper les Sons et Effets
```
Les adjectifs sont un peu redondants non ?